druid/src/widget/flex.rs

```diff
@@ -619,10 +619,14 @@ impl<T: Data> Widget<T> for Flex<T> {
         // these two are calculated but only used if we're baseline aligned
         let mut max_above_baseline = 0f64;
         let mut max_below_baseline = 0f64;
+        let mut any_use_baseline = matches!(self.cross_alignment, CrossAxisAlignment::Baseline);
```

Comment:
```suggestion
        let mut any_use_baseline = self.cross_alignment == CrossAxisAlignment::Baseline;
```

Reply:
What is the policy on `matches!` vs `==` given they will probably result in the same assembly.

Reply:
`matches!` is more verbose, but powerful. When I see it, I expect a more complicated enum comparison, probably ignoring some parts of the variant. If `==` does the job perfectly, using it makes more sense to me.

Reply:
I agree with this, I think if `PartialEq` is implemented and does the job that is clearer code.